modules/models/src/main/java/org/eclipse/xpanse/modules/models/view/ServiceDetailVo.java

```diff
@@ -27,4 +28,7 @@ public class ServiceDetailVo extends ServiceVo {
 
     @Schema(description = "The resource list of the deployed service.")
     private List<DeployResource> deployResources;
+
+    @Schema(description = "The property of the deployed service.")
```

Comment:
description -> "The properties of the deployed service."

Reply:
Ok